docs/dev/dev-mode.md

```diff
@@ -13,70 +13,100 @@ Description: Developer mode (dev mode) is intended to be used for testing and de
 
 Developer mode (dev mode) is intended to be used for testing and development purposes.
```

Comment:
Please help to add this one as an [attention text](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types):
```
!!! attention
    Developer mode (dev mode) is intended to be used for local testing and development purposes.
```

Reply:
updated